manifests/client.pp

```diff
@@ -153,13 +174,13 @@
       'xymon::client::monitor',
       $monitors,
       {
-        clientlaunch_config => $_clientlaunch_config,
-        files_path          => $_files_path,
-        xymon_user          => $xymon_user,
-        xymon_group         => $xymon_group,
-        xymon_service       => $service_name,
-        require             => Package[$package],
-        notify              => Service[$service_name]
+        clientlaunch_config   => $_clientlaunch_config,
+        files_path            => $_files_path,
+        xymon_user            => $xymon_user,
+        xymon_group           => $xymon_group,
+        xymon_service         => $service_name,
+        require               => Package[$package],
+        notify                => Service[$service_name]
```

Comment:
Hmm, did the auto formatter do this?

Reply:
Strange, right? I'd say the previous version was the correctly formatted one...

Reply:
Uh, i guess i had a longer parameter there for a short time, which got removed again. Will fix it, good catch!

Reply:
Fixed now.